app/controllers/contentful/static_controller.rb

```diff
@@ -5,7 +5,7 @@ def show
     if @model.present?
       track('static_page')
     else
-      render 'errors/not_found'
```

Comment:
I was thinking `redirect_to` is the better option. @ebrett what's your opinion?

Reply:
The redirect wasn't working, but also I wasn't sure if that was the right way to go. I don't think error 404 pages normally redirect you, as the incorrectly entered URL is normally still in the address bar

Reply:
yes, I agree with @agoldstone93 just render the message